test/orange/Orange.test.cc

```diff
@@ -502,6 +508,27 @@ TEST_F(FiveVolumesTest, params)
     EXPECT_FALSE(geo.supports_safety());
 }
 
+TEST_F(UniversesTest, params)
+{
+    const OrangeParams& geo = this->params();
+    EXPECT_EQ(9, geo.num_volumes());
+    EXPECT_EQ(21, geo.num_surfaces());
+    EXPECT_FALSE(geo.supports_safety());
+
+    EXPECT_VEC_SOFT_EQ(Real3({-2, -6, -1}), geo.bbox().lower());
+    EXPECT_VEC_SOFT_EQ(Real3({8, 4, 2}), geo.bbox().upper());
+
+    EXPECT_EQ("outer.[EXTERIOR]", geo.id_to_label(VolumeId{0}).name);
+    EXPECT_EQ("outer.inner_a", geo.id_to_label(VolumeId{1}).name);
+    EXPECT_EQ("outer.inner_b", geo.id_to_label(VolumeId{2}).name);
+    EXPECT_EQ("outer.bobby", geo.id_to_label(VolumeId{3}).name);
+    EXPECT_EQ("outer.johnny", geo.id_to_label(VolumeId{4}).name);
+    EXPECT_EQ("inner.[EXTERIOR]", geo.id_to_label(VolumeId{5}).name);
+    EXPECT_EQ("inner.a", geo.id_to_label(VolumeId{6}).name);
+    EXPECT_EQ("inner.b", geo.id_to_label(VolumeId{7}).name);
+    EXPECT_EQ("inner.c", geo.id_to_label(VolumeId{8}).name);
```

Comment:
This is all fine, but it might've been easier (and will be more robust in case the number of volumes is wrong) to loop over `range(VolumeId{geo.num_volumes()})` , `push_back` onto a vector of strings, and use `EXPECT_VEC_EQ`. (You can use the `PRINT_EXPECTED` macro to generate the initial list of expected values.)